lightningd/peer_htlcs.c

```diff
@@ -1384,6 +1384,7 @@ static void add_fulfill(u64 id, enum side side,
 }
 
 static void add_fail(u64 id, enum side side,
+		     enum onion_type malformed,
```

Comment:
I found the renaming from `failcode` to `malformed` rather hard to follow, could you add a comment to this?

Reply:
Actually, better: we should use the same terminology everywhere.  Let me create a patch...